src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.CoreCLR.cs

```diff
@@ -706,12 +703,41 @@ internal unsafe struct MethodTable
         [FieldOffset(ElementTypeOffset)]
         public void* ElementType;
 
+        /// <summary>
+        /// The PerInstInfo is used to describe the generic arguments and dictionary of this type.
+        /// It points as a PerInstInfo, which is an array of pointers to generic dictionaries, which then point
```

Comment:
> It points as a PerInstInfo 

This does not parse for me.

Reply:
I've updated the wording. Hopefully it is a bit more clear although for sure, the actual design of this is a very confusing structure.